src/moj/components/ticket-panel/README.md

```diff
@@ -0,0 +1,36 @@
+# Timeline
```

Comment:
@DanGuyMOJ are you able to transfer the context and research of this component here. Things like usage and use when sort of content